www/src/components/marketplace/MarketplaceSidebar.js

```diff
@@ -59,96 +52,127 @@ function MarketplaceSidebar(props) {
 
   function handleToggle(key, value) {
     const existing = searchParams.getAll(key)
+    const formatedValue = value.toLowerCase()
 
-    if (existing.includes(value)) {
+    if (existing.includes(formatedValue)) {
       setSearchParams({
-        ...searchParams,
-        [key]: existing.filter(v => v !== value),
+        ...Object.fromEntries(searchParams.entries()),
+        [key]: existing.filter(v => v !== formatedValue),
       })
     }
     else {
       setSearchParams({
-        ...searchParams,
-        [key]: [...existing, value],
+        ...Object.fromEntries(searchParams.entries()),
+        [key]: [...existing, formatedValue],
       })
     }
   }
 
   function isToggled(key, value) {
-    return searchParams.getAll(key).includes(value)
+    return searchParams.getAll(key).includes(value.toLowerCase())
+  }
+
+  function renderCategories() {
+    const sortedCategories = categoriesData.categories.slice().sort((a, b) => a.category.localeCompare(b.category))
+
+    return (
+      <Accordion
+        ghost
+        defaultExpanded
+        title={`Categories (${sortedCategories.length})`}
+        borderBottom="1px solid border !important"
+      >
+        {sortedCategories.map(({ category }) => (
+          <MarketplaceSidebarCheckbox
+            key={category}
+            toggled={isToggled('category', category)}
+            onClick={() => handleToggle('category', category)}
+            label={capitalize(category)}
+          />
+        ))}
+      </Accordion>
+    )
+  }
+
+  function renderPublishers() {
+    const sortedPublishers = ['Plural']
+
+    return (
+      <Accordion
+        ghost
+        title={`Publisher (${sortedPublishers.length})`}
+        borderBottom="1px solid border !important"
+      >
+        {sortedPublishers.map(publisher => (
+          <MarketplaceSidebarCheckbox
+            key={publisher}
+            toggled={isToggled('publisher', publisher)}
+            onClick={() => handleToggle('publisher', publisher)}
+            label={capitalize(publisher)}
+          />
+        ))}
+      </Accordion>
+    )
   }
 
-  const sortedCategories = categoriesData.categories.slice().sort((a, b) => a.category.localeCompare(b.category))
-  const sortedPublishers = ['Plural']
-  const sortedTags = tags.slice().sort((a, b) => a.tag.localeCompare(b.tag)).filter((x, i) => i < nDisplayedTags)
+  function renderTags() {
+    const sortedTags = tags.slice().sort((a, b) => a.name.localeCompare(b.name))
+    const fuse = new Fuse(sortedTags, searchOptions)
+    const resultTags = search ? fuse.search(search).map(({ item }) => item) : sortedTags.filter((x, i) => i < nDisplayedTags)
```

Comment:
we're definitely not going to be able to load all tags in one fetch in relatively short order, so this should really be moved to utilizing the gql apis properly

Reply:
I'm waiting on some green go from the backend team then.

Reply:
what do you need api wise?  I'm pretty sure we do have the capability to filter tags

Reply:
yep, you can use the `q` argument for the tags query to filter by a search string